ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -96,48 +108,59 @@ class CommandBarButton: UIButton {
 
     private let isPersistSelection: Bool
 
+    private var normalTintColor: UIColor {
+        return UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.foreground1])
+    }
+
     private var selectedTintColor: UIColor {
-        guard let window = window else {
-            return UIColor(light: Colors.communicationBlue,
-                           dark: .black)
+        guard let darkColorValue = fluentTheme.aliasTokens.colors[.foreground1].dark else {
+            return UIColor(light: UIColor(colorValue: fluentTheme.aliasTokens.colors[.brandForeground4].light),
+                           dark: UIColor(colorValue: GlobalTokens.neutralColors(.white)))
         }
 
-        return UIColor(light: Colors.primary(for: window),
-                       dark: .black)
+        return UIColor(light: UIColor(colorValue: fluentTheme.aliasTokens.colors[.brandForeground4].light),
+                       dark: UIColor(colorValue: darkColorValue))
     }
 
     private var selectedBackgroundColor: UIColor {
-        guard let window = window else {
-            return UIColor(light: Colors.Palette.communicationBlueTint30.color,
-                           dark: Colors.Palette.communicationBlue.color)
+        guard let darkColorValue = fluentTheme.aliasTokens.colors[.background5Selected].dark else {
```

Comment:
Idea for simplifying this code a bit, let me know which you prefer:

```swift
let darkColorValue = fluentTheme.aliasTokens.colors[.background5Selected].dark ?? GlobalTokens.neutralColors(.grey38)
return UIColor(light: UIColor(colorValue: fluentTheme.aliasTokens.colors[.brandBackground4].light),
               dark: UIColor(colorValue: darkColorValue))
```

Reply:
I can actually bypass unwrapping the value with the `DynamicColor` initializer so I'll use that instead.